docker/Dockerfile

```diff
@@ -40,25 +29,14 @@ RUN INFER_VERSION=v0.12.1; \
     ln -s ${PWD}/infer-linux64-$INFER_VERSION /infer
 
 # Compile Infer
-RUN OCAML_VERSION=4.04.2; \
-    opam init --compiler=$OCAML_VERSION -j $(getconf _NPROCESSORS_ONLN || echo 1) --yes && \
-    cd /infer && \
-    ./build-infer.sh --opam-switch $OCAML_VERSION && \
-    rm -rf /root/.opam
+RUN OCAML_VERSION=4.05.0; \
```

Comment:
Building infer with flambda makes its backend ~25% faster but worsen build times (f8d7c810452ce3a4d2e7027e38f5d00426a2a917). I think it's worth turning on by default, like we do in the in-tree build:
```
RUN OCAML_VERSION=4.05.0+flambda; \
```

Reply:
Sure, will fix this.